packages/SystemsThinkingWorkbench-Core.package/STWTransferConnection.class/instance/performSimulationStep.st

```diff
@@ -1,12 +1,15 @@
 updating
 performSimulationStep
-	
+
 	| particle |
 	
-	self startNode level: self startNode level - self intake.
-	particle := STWParticle new center: (self position + self segment start); color: self startNode fillingMorph color.
-	self particles add: particle.
-	self addMorph: particle.
-	self lastStepTime ifNil: [
-		self startStepping.
-		self lastStepTime: DateAndTime millisecondClockValue.].
+	((self startNode canHoldWithout: self intake)
```

Comment:
This might be a little nitpicky, but maybe the 
`(self startNode canSupply: self intake) and: (self endNode canReceive: self output)` could be moved to a seperate method, like `canPerformTransaction`, what do you think?